avm/res/recovery-services/vault/backup-fabric/protection-container/main.bicep

```
@@ -0,0 +1,86 @@
+metadata name = 'Recovery Services Vault Protection Container'
+metadata description = 'This module deploys a Recovery Services Vault Protection Container.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Conditional. The name of the parent Azure Recovery Service Vault. Required if the template is used in a standalone deployment.')
+param recoveryVaultName string
+
+@description('Required. Name of the Azure Recovery Service Vault Protection Container.')
+param name string
+
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Backup management type to execute the current Protection Container job.')
+@allowed([
+  'AzureBackupServer'
+  'AzureIaasVM'
+  'AzureSql'
+  'AzureStorage'
+  'AzureWorkload'
+  'DPM'
+  'DefaultBackup'
+  'Invalid'
+  'MAB'
+  ''
+])
+param backupManagementType string = ''
+
+@description('Optional. Resource ID of the target resource for the Protection Container.')
+param sourceResourceId string = ''
```

```suggestion
param sourceResourceId string?
```

same as per `backupManagementType`